code/index.html

```diff
@@ -1,17 +1,150 @@
+          alt="A woman in the tennis court">
+        <h3>Adult Camp</h3>
+        <p>The adult camps are structured to help players excel in matches, leagues, tournaments, and competitions.
+          Participants train with the Ace Tennis Academy’s skilled coaches, employing the Ace Tennis Academy Methodology
+          for comprehensive growth. The camps focus on advanced drills, tactical insights, and mental conditioning, all
+          designed to elevate performance. These programs provide a challenging yet supportive environment where adults
+          can enhance their skills, build confidence, and achieve their personal best.
+        </p>
+        <button type="submit" class="btn-create">Learn more</button>
+      </div>
+      <div class="grid-item">
+        <img
+          src="https://scontent-arn2-1.xx.fbcdn.net/v/t39.30808-6/457740804_8505520106149554_7251837137325307767_n.jpg?_nc_cat=100&ccb=1-7&_nc_sid=127cfc&_nc_ohc=Gb1PV5cs9R4Q7kNvgGBgzub&_nc_ht=scontent-arn2-1.xx&cb_e2o_trans=t&oh=00_AYDqtpeJKrZR8sEsYkrHS82sg4kUM-yqVcP12U8zWtiGXQ&oe=66DA38DA"
+          alt="Woman in the tennis court">
+        <h3>Weekend Camp</h3>
+        <p>Enjoy a weekend of high-intensity tennis training! Improve your skills while having fun, whether solo, with
+          friends, or with family, at the Ace Tennis Academy Resort. Experience intensive coaching sessions, dynamic
+          drills, and matches, combined with relaxing amenities, in a beautiful setting. This unique getaway blends
+          sport and leisure, giving you the chance to train hard, relax under the sun, and create unforgettable
+          memories. Perfect for tennis lovers seeking both improvement and enjoyment on the court.</p>
+        <button type="submit" class="btn-create">Learn more</button>
+      </div>
+
+  </section>
+
+  <!-- Footer Section -->
+  <footer class="footer">
+    <div class="footer-container">
+      <div class="footer-logo">
+        <h1>Ace Tennis Academy</h1>
+      </div>
+      <div class="footer-links">
+        <a href="#">About</a>
+        <a href="#">Privacy Policy</a>
+        <a href="#">Terms of Service</a>
+        <a href="#">Contact</a>
+      </div>
+      <div class="footer-social">
+        <a href="#"><img src="https://cdn-icons-png.flaticon.com/128/20/20837.png" alt="Facebook"></a>
+        <a href="#"><img src="https://cdn-icons-png.flaticon.com/128/733/733635.png" alt="Twitter"></a>
+        <a href="#"><img src="https://cdn-icons-png.flaticon.com/128/1400/1400829.png" alt="Instagram"></a>
+      </div>
+      <p class="footer-text">&copy; 2024 Ace Tennis Academy. All rights reserved.</p>
+    </div>
+  </footer>
+
+</body>
 
-  </body>
 </html>
+
+</body>
+
+</html>
+
+
+</html>
```

Comment:
I like the overall design of the site, especially in the biggest screen size. I think the use of boxes is really good because it makes the site but also the code easier to overlook. I think the big image and the transparent quote-box is a nice way to combine image and text together.